src/data-types.js

```diff
@@ -87,6 +87,18 @@ export function getTypes(options) {
 		}
 	}
 
+	var int64 = {
+		unpack: function(buffer){
+			var value = buffer.readUInt64LE(buffer.pointer);
```

Comment:
npm install doesn't actually run the code, it just installs any dependencies for the project. 

Since Javascript is a scripting language, there is no pre-compilation step before being able to run the project (although V8 attempts to compile JIT), hence why you didn't get an error.

On 08/06/2015, at 10:37 PM, "UserMcUser" <notifications@github.com> wrote:

> In src/data-types.js:
> 
> > @@ -87,6 +87,18 @@ export function getTypes(options) {
> >  		}
> >  	}
> >  
> > +	var int64 = {
> > +		unpack: function(buffer){
> > +			var value = buffer.readUInt64LE(buffer.pointer);
> When I ran 'npm install' it finished without error. I admit I did no additional testing of any kind.
> 
> —
> Reply to this email directly or view it on GitHub.
>